apps/dash-multipage-report/app.py

```diff
@@ -0,0 +1,2738 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
+import dash_table
+import pandas as pd
+import lorem
+import pathlib
+
+# Path
+BASE_PATH = pathlib.Path(__file__).parent.resolve()
+DATA_PATH = BASE_PATH.joinpath("Data").resolve()
+
+## Read in data
+supplyDemand = pd.read_csv(DATA_PATH.joinpath("supplyDemand.csv"))
+actualSeasonal = pd.read_csv(DATA_PATH.joinpath("actualSeasonal.csv"))
+industrailProd = pd.read_csv(DATA_PATH.joinpath("industrailProd.csv"))
+globalMarket = pd.read_csv(DATA_PATH.joinpath("globalMarket.csv"))
+oecdCommersial = pd.read_csv(DATA_PATH.joinpath("oecdCommersial.csv"))
+wtiPrices = pd.read_csv(DATA_PATH.joinpath("wtiPrices.csv"))
+epxEquity = pd.read_csv(DATA_PATH.joinpath("epxEquity.csv"))
+chinaSpr = pd.read_csv(DATA_PATH.joinpath("chinaSpr.csv"))
+oecdIndustry = pd.read_csv(DATA_PATH.joinpath("oecdIndustry.csv"))
+wtiOilprices = pd.read_csv(DATA_PATH.joinpath("wtiOilprices.csv"))
+productionCost = pd.read_csv(DATA_PATH.joinpath("productionCost.csv"))
+production2015 = pd.read_csv(DATA_PATH.joinpath("production2015.csv"))
+energyShare = pd.read_csv(DATA_PATH.joinpath("energyShare.csv"))
+adjustedSales = pd.read_csv(DATA_PATH.joinpath("adjustedSales.csv"))
+growthGdp = pd.read_csv(DATA_PATH.joinpath("growthGdp.csv"))
```

Comment:
Maybe you can put these csv files in a remote github repo instead of keeping it in this folder to save some space.

Reply:
@Scott-Clone Let's open an issue/PR and do as @ronniebugia suggests _after_ this one lands.